### DISCUSSION TITLE
[Destination PostgreSQL] docker freezing at normalization step

### DISCUSSION THREAD

**Discussion by gtncc** [airbytehq/airbyte#34257] [Connector Questions]
Hello everyone.

I'm coming here because I have a problem at the normalization step. The job freeze. I tried disabling the normalization step and makes the job work without any problem.

I'm stuck at the state where the created table is containing the 3 columns:

- _airbyte_ab_id
- _airbyte_data
- _airbyte_emitted_at

Do you have anything regarding this issue? Here is the log file:
[airbyte_mssql_postgres.txt](https://github.com/airbytehq/airbyte/files/13938103/airbyte_mssql_postgres.txt)

Here a screenshot of the error:
<img width="599" alt="Capture d’écran 2024-01-15 à 17 05 29" src="https://github.com/airbytehq/airbyte/assets/99823484/08b7d806-d337-4079-8693-996f65a7aabd">

Thank you !

**Comment by marcosmarxm** [comment]
Can you monitor the resources? Maybe you need to increase mem/cpu to run normalization.
Normalization is a deprecated feature so probably won't receive any change in the future until Postgres be upgraded to Destination V2 mode.

**Reply by gtncc** [reply]
Thank you for your answer ! I will try it and go back to you. So I should normalize the data myself?

**Reply by marcosmarxm** [reply]
You have the option to export the normalization project and use as custom transformation or run directly using dbt.
More information you can read [here](https://docs.airbyte.com/operator-guides/transformation-and-normalization/transformations-with-sql). Doing this you can maybe reduce size of what its going to be normalized and use less resources.

[a comment by an automated account is not shown]
